### ISSUE TITLE
accessibility tests from Wave and Axe give many error's

### ISSUE THREAD

**Issue by cobra779** [aldabil21/react-scheduler#192]
![wavetest](https://user-images.githubusercontent.com/73694342/232781980-f2eecc2f-04e3-4c6b-93f0-f5d0395bcf48.png)
![axetest](https://user-images.githubusercontent.com/73694342/232782064-d78d9b57-dd97-4580-b88a-5f6d202868f2.png)

**Comment by aldabil21** [comment]
I believe this is a missing `aria-label` of the buttons, I will add that asap
